web/packages/teleport/src/services/userEvent/types.ts

```diff
@@ -81,6 +81,9 @@ export enum DiscoverEvent {
   DesktopActiveDirectoryToolsInstall = 'tp.ui.discover.desktop.activeDirectory.tools.install',
   DesktopActiveDirectoryConfigure = 'tp.ui.discover.desktop.activeDirectory.configure',
   AutoDiscoveredResources = 'tp.ui.discover.autoDiscoveredResources',
+  EC2InstanceSelection = 'tp.ui.discover.selectedEC2Instance',
+  EC2DeployEICE = 'tp.ui.discover.deployEICE',
+  EC2CreateNode = 'tp.ui.discover.createNode',
```

Comment:
:nail_care: 
I think it is clearer because the event name is only `createNode`.
Up to you :+1: 
```suggestion
  CreateNode = 'tp.ui.discover.createNode',
```